health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -1,44 +1,360 @@
+              />
+            </>
+          );
+      }
+    },
+  },
+  MicroplanCampaignSearchConfig: {
+    preProcess: (data, additionalDetails) => {
+      const searchParams = new URLSearchParams(window.location.search);
+      const userId = searchParams.get("userId");
+      const status = searchParams.get("status");
+      userId&&(data.body.PlanConfigurationSearchCriteria.userUuid = userId);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)